src/components/bookshelf.tsx

```
@@ -66,18 +76,63 @@ export const Bookshelf = () => {
       .catch(error => console.error(`There was an error creating the ${title} book: ${error}`))
   }
 
-  // Submit new book
-  const handleBookSubmit = () => {
-    // Check if all fields are filled
-    if (author.length > 0 && title.length > 0 && pubDate.length > 0 && rating.length > 0) {
-      // Create new book
-      handleBookCreate()
+  // Edit book
+  const bookEdit = (id: number) => {
+    // Send PUT request to 'books/edit' endpoint
+    axios
+      .put('http://localhost:4001/books/edit', {
+        id: id,
+        author: author,
+        title: title,
+        pubDate: pubDate,
+        rating: rating,
+        genre: genre
+      })
+      .then(() => {
+        console.log(`Book ${title} edited.`)
 
-      console.info(`Book ${title} by ${author} added.`)
+        // Fetch all books to refresh
+        // the books on the bookshelf list
+        fetchBooks()
+      })
+      .catch(error => console.error(`There was an error editing the ${title} book: ${error}`))
```

in situations like this it is best to raise a more visible alert, i.e. red error text shown to user, or an alert vs silently failing.